### PR TITLE
added nodejs aliases

### PR DESCRIPTION
**Pull request by tbhaxor**
Added the most frequently used commands used be nodejs developers

### CODE REVIEW

File: aliases/available/node.aliases.bash [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cite 'about-alias'
+about-alias 'the nodejs environment aliases'
+
+# alias to setup nodejs development environment
+alias node-dev='export NODE_ENV=development'
+
+# alias to setup nodejs production environment
+alias node-prod='export NODE_ENV=production'
```
Comment (nwinkler) [inline aliases/available/node.aliases.bash]: Please add a line break at the end of the file.

File: aliases/available/node.aliases.bash [hunk 2]
```diff
@@ -0,0 +1,8 @@
+cite 'about-alias'
+about-alias 'the nodejs environment aliases'
```
Comment (nwinkler) [inline aliases/available/node.aliases.bash]: Please change to "Node.js", which is the official name...

File: aliases/available/node.aliases.bash (hunk 1 above)
Comment (tbhaxor) [reply aliases/available/node.aliases.bash]: Done

File: aliases/available/node.aliases.bash (hunk 2 above)
Comment (tbhaxor) [reply aliases/available/node.aliases.bash]: Done

**Comment by nwinkler** [comment]
Thanks

**Pull request merged**